.github/workflows/nuget-publish.yml

```diff
@@ -0,0 +1,70 @@
+name: NuGet Publish
+
+on:
+  push:
+    branches: [ master ]
+
+env:
+  NUGET_DIR: ${{ github.workspace }}/nuget
+  NUGET_ARTIFACT: nuget
+
+jobs:
+  testing:
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    uses: ./.github/workflows/test.yml
+
+  pack: 
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v3
```

Comment:
Can we pin dotnet to specific version, similar to https://github.com/ButterCMS/buttercms-ruby/pull/28/files#diff-1ac2730f9bc3ac91b4137899948628a3b5c6b181763cbfb93fa9d979bb6c8ac5R17-R19?

Reply:
Resolved in https://github.com/ButterCMS/buttercms-csharp/pull/40/commits/f3d2d5e3f00c7bfcdd3405cd42a7cfb56b556a32